README.md

```diff
@@ -15,6 +15,24 @@ This integrates the two with each other. Just click on a \*.kdbx file in Your Ne
 
 To update to a new version, simply repeat these steps.
 
+## Mimetype detection
+
+Unfortunately, apps can't declare new mimetypes on the fly. To make
+Keeweb work properly, you need to add a new mimetype in the
+`mimetypemapping.json` file (at Nextcloud level).
```

Comment:
"at Nextcloud level" seems confusing, I don't know what this wants to tell me.

Reply:
I just have taken the description as it is from Ownpad. The section `(at Nextcloud level)`can be removed completely. I'll changed that.